Keen.NetStandard/KeenClient.cs

```diff
@@ -972,5 +973,177 @@ public JObject Query(string queryName, Dictionary<string, string> parms)
                 throw ex.TryUnwrap();
             }
         }
+
+        /// <summary>
+        /// Get query results from a Cached Dataset.
+        /// </summary>
+        /// <param name="datasetName">Name of cached dataset to query.</param>
+        /// <param name="indexBy">The string property value you want to retrieve results by.</param>
+        /// <param name="timeframe">Limits retrieval of results to a specific portion of the Cached Dataset</param>
+        /// <returns>An instance of Newtonsoft.Json.Linq.JObject containing query results and metadata defining the cached dataset.</returns>
+        public async Task<JObject> QueryDatasetAsync(string datasetName, string indexBy, string timeframe)
+        {
+            return await Datasets.GetResultsAsync(datasetName, indexBy, timeframe);
```

Comment:
This needs to call `.ConfigureAwait(false)`

Reply:
Will do here and elsewhere. I fixed some down in the Datasets class but missed these 😞